drivers/docker/driver_test.go

```diff
@@ -1004,6 +1004,34 @@ func TestDockerDriver_SecurityOpt(t *testing.T) {
 	require.Exactly(t, cfg.SecurityOpt, container.HostConfig.SecurityOpt)
 }
 
+func TestDockerDriver_SecurityOptFromFile(t *testing.T) {
+	seccompPath := "./test-resources/docker/seccomp.json"
+
+	if runtime.GOOS == "windows" {
```

Comment:
You can also move the test to `driver_unix_test.go` (or `driver_linux_test.go`) instead if it doesn't apply to Windows.  Does it apply to macOS?

Reply:
Honestly not sure. I copied the boiler plate from the existing seccomp=unconfined test. I would have thought even windows would run the container in a VM where seccomp applies but I have no good way to test.

Reply:
Makes sense - I'll test behavior!